projects/lexical_shared/lib/lexical/formats.ex

```diff
@@ -68,8 +68,14 @@ defmodule Lexical.Formats do
     string_name = Atom.to_string(module_name)
 
     if String.contains?(string_name, ".") do
-      module_name
-      |> Module.split()
+      module_part =
+        case string_name do
+          "Elixir." <> rest -> rest
+          other -> other
+        end
+
+      module_part
+      |> String.split()
```

Comment:
I'm not sure I'm understanding what this is handling. It's splitting on whitespace, which should never be in the module name, right? So this ends up being a list of one element, and then the join just unwraps the element.

Reply:
I'm working in a codebase where module names end up like :"Foo.Bar.Baz", it's very wrong, but legal.

Reply:
Oh, never mind, I got ya.